apps/choleskey/generateInputMatrix.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This is an Jupyter Script for generating the Input Positive Definition(PD) Matrix for Cholesky decomposition.
+# The generated .txt file stores the entire PD matrix with the matrix elems row by row. 
+# This input.txt file can be used directly In benchmark part in each cpp file.
+
+
+import numpy as np
+
+#enter matrix size
+matrixSize = 4
+
+#1. Generates a random matrix A of size matrixSize x matrixSize.
+A = np.random.rand(matrixSize, matrixSize)
```

Comment:
I would recommend not generating matrices using rand here as the probability of generating a wrong matrix may get increasingly higher for larger matrices. Instead I would recommend looking into methods to generate positive definitive matrices with high probability such as translating this matlab code to python: https://math.stackexchange.com/questions/357980/how-to-generate-random-symmetric-positive-definite-matrices-using-matlab

Reply:
agree, I have another helper function in tiled_cholesky implementation which can generate the pd matrix using blas lib. 
I suggest to leave this PR alone at this moment at this moment.
And we can close this pr when get the better method.